controllers/storagecluster/storageconsumer.go

```diff
@@ -109,6 +111,16 @@ func (s *storageConsumer) ensureDeleted(_ *StorageClusterReconciler, _ *ocsv1.St
 	return ctrl.Result{}, nil
 }
 
+func getExternalStorageClassBlaclistSelector() labels.Selector {
+	blackListRequirement, err := labels.NewRequirement(util.ExternalClassLabelKey, selection.NotEquals, []string{"true"})
+	if err != nil {
+		fmt.Println(err)
```

Comment:
If you want to print, use a log instead of fmt

Reply:
I think we need to error it, use fmt.error